.github/workflows/ci.yaml

```diff
@@ -49,9 +49,18 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           prettier: true
-          eslint: false
+          eslint: true
           eslint_args: '--max-warnings 0'
           eslint_extensions: js,jsx,ts,tsx
 
+      - name: Build library
+        run: npm run build
+      - name: Build examples/create-react-app
+        run: cd examples/create-react-app && npm run build
+      - name: Build examples/nextjs
+        run: cd examples/nextjs && npm run build
+      - name: Build examples/typescript
+        run: cd examples/typescript && npm run build
```

Comment:
__*Nice to have*__

Might not be bad to add a `build:all` script in the root json that can just do all of these in one go

Reply:
I wrote a script for this in ts and then figtured I need ts-node to run it. Since `packages.json` deps are broken, I'd rather do it in a separate PR after we fix deps.